rules/prometheus-flashsystem-rules.yaml

```diff
@@ -8,113 +8,171 @@ metadata:
+          for: 5m
+          labels:
+            severity: warning
+    - name: flashsystem-pool-status-alert.rules
+      rules:
+        - alert: FlashsystemPoolWarningState
+          annotations:
+            description: Subsystem storage "{{$labels.subsystem_name}}", pool "{{$labels.pool_name}}" is in degraded status for more than 5m.
+            message: 'Subsystem storage pool "{{$labels.pool_name}}" is in degraded state.'
+            severity_level: warning
+            storage_type: ibm_flashsystem
+          expr: |
+            flashsystem_pool_health{subsystem_name="flashsystem-subsystem-name-template"} ==1
+          for: 5m
+          labels:
+            severity: warning
+        - alert: FlashsystemPoolErrorState
+          annotations:
+            description: Subsystem storage "{{$labels.subsystem_name}}", pool "{{$labels.pool_name}}" is offline.
+            message: 'Subsystem storage pool "{{$labels.pool_name}}" is in offline state.'
+            severity_level: error
+            storage_type: ibm_flashsystem
+          expr: |
+            flashsystem_pool_health{subsystem_name="flashsystem-subsystem-name-template"} ==2
+          for: 1m
+          labels:
+            severity: critical
+    - name: flashsystem-pool-utilization-alert.rules
+      rules:
+        - alert: FlashSystemPoolPhysicalCapacityNearFull
+          annotations:
+            description: Subsystem storage "{{$labels.subsystem_name}}", pool "{{$labels.pool_name}}" physical capacity utilization has crossed warning threshold for more than 5m.
+            message: 'Subsystem storage "{{$labels.subsystem_name}}" on pool "{{$labels.pool_name}}" physical capacity is nearing full. Data deletion or pool expansion is required.'
+            severity_level: warning
+            storage_type: ibm_flashsystem
+          expr: |
+            flashsystem_pool_capacity_used_bytes{subsystem_name="flashsystem-subsystem-name-template"}/(flashsystem_pool_capacity_usable_bytes + flashsystem_pool_capacity_used_bytes) > flashsystem_pool_capacity_warning_threshold/100
+          for: 5m
+          labels:
+            severity: warning
+        - alert: FlashSystemPoolLogicalCapacityNearFull
+          annotations:
+            description: Subsystem storage "{{$labels.subsystem_name}}", pool "{{$labels.pool_name}}" logical capacity utilization has crossed warning threshold for more than 5m.
+            message: 'Subsystem storage "{{$labels.subsystem_name}}" on pool "{{$labels.pool_name}}" logical capacity is nearing full. Data deletion or pool expansion is required.'
+            severity_level: warning
+            storage_type: ibm_flashsystem
+          expr: |
+            flashsystem_pool_logical_capacity_used_bytes{subsystem_name="flashsystem-subsystem-name-template"}/(flashsystem_pool_logical_capacity_usable_bytes + flashsystem_pool_logical_capacity_used_bytes) > flashsystem_pool_capacity_warning_threshold/100
```

Comment:
I think we need the `subsystem_name` filter also for `flashsystem_pool_logical_capacity_usable_bytes` and `flashsystem_pool_logical_capacity_used_bytes`, no ?